src/pages/search/audio.vue

```diff
@@ -1,24 +1,75 @@
-<!-- noresult is hardcoded since it does not yet support built-in audio search -->
 <template>
-  <div id="tab-audio" role="tabpanel" aria-labelledby="audio">
-    <AudioResultsList v-if="supported" @onLoadMoreAudios="onLoadMoreAudios" />
-    <MetaSearchForm type="audio" :noresult="false" :supported="supported" />
-  </div>
+  <section>
+    <template v-if="supported">
+      <AudioTrack
+        v-for="audio in mediaResults.items"
+        :key="audio.id"
+        class="px-6 mb-6"
+        :audio="audio"
+        :size="audioTrackSize"
+        layout="row"
+      />
+
+      <template v-if="isError" class="m-auto w-1/2 text-center pt-6">
+        <h5>{{ errorHeader }}</h5>
+        <p>{{ fetchState.fetchingError }}</p>
+      </template>
+      <LoadMoreButton
+        v-if="shouldShowLoadMore"
+        :is-error="isError"
+        :is-fetching="fetchState.isFetching"
+        :is-finished="fetchState.isFinished"
+        media-type="audio"
+        data-testid="load-more"
+        @onLoadMore="onLoadMore"
+      />
+    </template>
+  </section>
 </template>
 
 <script>
 export default {
   name: 'AudioSearch',
-  data() {
-    return {
-      // Only show audio results if non-image results are supported
-      supported: process.env.enableAudio,
-    }
+  props: {
+    mediaResults: {},
+    fetchState: {},
+    isFilterVisible: {},
```

Comment:
Should we add prop documentation for these?

Reply:
I tried adding the prop types, @sarayourfriend , and ended up converting the search pages to use composition API, and creating a new composable.  So, now I'm scared to merge this without having someone look at the PR again...

Reply:
I rebased on the main to check with `npm types`, and the composable passes the type check!!! 
By the way, maybe you have the audio set as unsupported? That would probably cause the e2e tests to fail as there wouldn't be an Audio tab.

Reply:
> the composable passes the type check

Nice! That's awesome 🤩